projects/or_scs/src/Mirza/OrgRegistry/Types.hs

```diff
@@ -72,7 +72,10 @@ orContextMinimal :: EnvType -> Pool Connection -> K.LogEnv -> K.LogContexts -> K
 orContextMinimal a b c d e = ORContextGeneric a b c d e () ()
 
 orContextComplete :: ORContextMinimal -> Audience -> JWK-> ORContextComplete
-orContextComplete (ORContextGeneric a b c d e () ()) f g = ORContextGeneric a b c d e f g
+orContextComplete (ORContextGeneric envT poolConn logEnv logCtxs nameSpace () ()) audList@(Audience strUriList) jwk
+    = if (null strUriList)
```

Comment:
I'm in two minds here,

I immediately thought that we should be using a NonEmpty list rather than doing a check here, but then I realised that the types we are using come from Crypto.JWT.

So it's either define our own types and convert a `NonEmpty StringOrUri` to an `Audience` closer to the JWT verify call site, or we leave it...

Reply:
If it's used any more than it already is, it's worth using our custom types.

Reply:
Yeah I think Pauls solution here would have been cleaner and better then polluting those two pure functions.